### PR TITLE
Remove uneeded gtest dependency, build with verbosely with CMake & ni…

### PR DESCRIPTION
**Pull request by larroy**
…nja.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -270,10 +270,8 @@ try {
             -DUSE_CUDNN=1              \
             -DCMAKE_BUILD_TYPE=Release \
             """
-            def flag = """             \
-            -j\$(nproc)
-            """
-          cmake("build_cuda", defines, flag)
+            def flag = "-v"
```
Comment (marcoabreu) [inline Jenkinsfile]: Is the number of build threads automatically determined?

Comment (larroy) [reply Jenkinsfile]: @marcoabreu affirmative

**Comment by marcoabreu** [comment]
As explained offline, changes on the dockerfile regarding Nvidia drivers must not be tested in PRs on the prod system. closing this PR.

**Comment by larroy** [comment]
@marcoabreu it passed CI, I would like to merge this. And I don't see how can it affect anything on the prod system.

**Pull request closed without merging**